sanic/app.py

```diff
@@ -1067,7 +1069,7 @@ def run(
         stop_event: Any = None,
         register_sys_signals: bool = True,
         access_log: Optional[bool] = None,
-        **kwargs: Any,
```

Comment:
Why remove this? Breaking change, no?

Reply:
It was only used for `auto_reload`, which is now a named argument, and it was causing invisible bugs as any typoed kwargs would be silently ignored. As far as I know, there are not other named parameters that should be caught here, so it will only break applications with such typos in their args. Notice that the `*` is also a potentially breaking change for anyone who tried to provide `debug` as positional rather than by keyword.